resources/views/profile/overview.blade.php

```diff
@@ -0,0 +1,51 @@
+@extends('layouts.base')
+@props(['orders'])
+
+@php
+    $title = __('order_tracking.order_tracking_title');
+@endphp
+
+@section('content')
+    <div class="breadcrumbs-container">
+        <ol class="flex items-center whitespace-nowrap" aria-label="Breadcrumb">
+            <li class="inline-flex items-center">
+                <a href="{{ route('home') }}"
+                    class="flex items-center text-sm text-gray-500 hover:text-blue-600 focus:outline-none focus:text-blue-600 dark:focus:text-blue-500">
+                    <svg class="flex-shrink-0 me-3 size-4" xmlns="http://www.w3.org/2000/svg" width="24" height="24"
+                        viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round"
+                        stroke-linejoin="round">
+                        <path d="m3 9 9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z" />
+                        <polyline points="9 22 9 12 15 12 15 22" />
+                    </svg>
+                    {{ __('navbar.home') }}
+                </a>
+                <svg class="flex-shrink-0 mx-2 overflow-visible size-4 text-gray-400 dark:text-neutral-600"
+                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"
+                    stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                    <path d="m9 18 6-6-6-6" />
+                </svg>
+            </li>
+            <li class="inline-flex items-center">
+                <a href="{{ route('profile.index')}}"
+                    class="flex items-center text-sm text-gray-500 hover:text-blue-600 focus:outline-none focus:text-blue-600 dark:focus:text-blue-500">
+                    {{ __('auth/profile.profile')}}
+                </a>
+                <svg class="flex-shrink-0 mx-2 overflow-visible size-4 text-gray-400 dark:text-neutral-600"
+                    xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none"
+                    stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+                    <path d="m9 18 6-6-6-6" />
+                </svg>
+            </li>
+            <li class="inline-flex items-center text-sm font-semibold text-gray-800 truncate dark:text-gray-200"
+                aria-current="page">
+                {{ __('auth/profile.overview_orders') }}
+            </li>
+        </ol>
+    </div>
+
+    <div class="shadow p-4 m-4 rounded-xl flex flex-col gap-4">
+        @foreach ($orders as $order)
+            <x-order-preview :order="$order" />
+        @endforeach
```

Comment:
Kan je hier alvast ervoor zorgen dat het een link is die naar # leidt? Net zoals in edit.blade.php.